test/e2e/v2/plugin_cluster_test.go

```diff
@@ -39,8 +39,9 @@ var _ = Describe("kubebuilder", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(kbc.Prepare()).To(Succeed())
 
+			// Install cert-manager with v1beta2 CRs.
 			By("installing cert manager bundle")
-			Expect(kbc.InstallCertManager()).To(Succeed())
+			Expect(kbc.InstallCertManager(true)).To(Succeed())
```

Comment:
It shows a little strange we have a func `InstallCertManager` which receives a boolean. WDYT about `InstallCertManager(version string)` or `InstallCertManagerV2` and `InstallCertManagerV3`

Reply:
`InstallCertManager(hasv1beta2CRs bool)` is plugin-agnostic, so when a go/v4 plugin is added, the e2e test can use `InstallCertManager(false)` like go/v3 does instead of adding a `InstallCertManagerV4()` method.

Reply:
WDYT about `InstallCertManager(version string)`? the `InstallCertManager(hasv1beta2CRs bool)` approach shows harder to get how it works.

Reply:
What version does `version` correspond to? cert-manager? Exporting the cert-manager version is probably not great since the URL chosen depends on which version you pass. Realistically there are only two options: use an "old" bundle URL for an "old" version, or use the "new" bundle URL (depending on cluster type) for a "new" version. This is why I chose a boolean, in addition to not wanting to expose `InstallCertManager()` internals. If `version` is decided to be the best parameter, I suggest requiring a full url, like `InstallCertManager(url string)`.

Reply:
It is a little confusing. Maybe `InstallCertManagerWithV1beta1`. However, I a strangle to find a way to make it more clear as well by only renaming it. So, let's move forward with it we can do a follow up to improve the understandability/maintability always.